openshift-ci/Dockerfile.tools

```diff
@@ -9,7 +9,7 @@ ENV LANG=en_US.utf8 \
     PATH=$PATH:$GOPATH/bin \
     GIT_COMMITTER_NAME=devtools \
     GIT_COMMITTER_EMAIL=devtools@redhat.com \
-    GOLANG_VERSION=go1.20.11 \
+    GOLANG_VERSION=go1.21.13 \
     GOLANG_SHA256=ef79a11aa095a08772d2a69e4f152f897c4e96ee297b0dc20264b7dec2961abe
```

Comment:
you need to update also the expected SHA256

Reply:
done in [22393a7](https://github.com/codeready-toolchain/toolchain-e2e/pull/1097/commits/22393a7a03f31cd153f671048a552a9bbf4e5df5)